Stride.BepuPhysics/Processors/ContainerProcessor.cs

```diff
@@ -31,43 +34,12 @@ protected override void OnEntityComponentAdding(Entity entity, [NotNull] Contain
 
             component.ContainerData = new(component, _bepuConfiguration, _game);
             component.ContainerData.RebuildContainer();
-            var parent = GetComponentsInParents<ContainerComponent>(entity).FirstOrDefault();
-            if (parent != null)
-            {
-                parent.ContainerData?.RebuildContainer();
-            }
```

Comment:
I think this stuff was for the previous version of colliders ? Where they add to add themselves to their container. Roll that back if I'm mistaken